client/src/app/shared/buttons/action-dropdown.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="dropdown-root" ngbDropdown [placement]="placement" *ngIf="areActionsDisplayed(actions, entry)">
+<div class="dropdown-root" ngbDropdown [placement]="placement" container="body" autoClose="outside" *ngIf="areActionsDisplayed(actions, entry)">
```

Comment:
```suggestion
<div class="dropdown-root" ngbDropdown [placement]="placement" container="body" *ngIf="areActionsDisplayed(actions, entry)">
```

You cannot use `autoClose="outside"`: the action-dropdown then doesn't close on menu clicks, which is required in some scenarios (coming to mind, the addition of a video in a playlist, where the previous dropdown is kept open as the new one opens…).